src/utils/Downloader.js

```diff
@@ -135,10 +134,11 @@ class Downloader {
       let res = {};
       try {
         res = await this._client(uri, options);
+        const body = await res.text();
```

Comment:
Why introduce a variable?

Reply:
to make sure the response body is always consumed, i.e. independent of `res.ok`.

Reply:
Got it, thanks.